Workbooks/Dns.json

```diff
@@ -159,7 +161,7 @@
       "type": 3,
       "content": {
         "version": "KqlItem/1.0",
-        "query": "DnsEvents\r\n| extend IPAddresses = iif(IPAddresses==\"\", \"empty\", IPAddresses)\r\n| where \"{IpAddresses:lable}\" == \"All\" or IPAddresses in ({IpAddresses})\r\n| extend type = case(\r\n    QueryType == \"A\", \"IPv4 address record\", \r\n    QueryType == \"AAAA\", \"IPv6 address record\",\r\n    QueryType == \"ANY\", \"All cached records\",\r\n    QueryType == \"CNAME\", \"Canonical name record\",\r\n    QueryType == \"MX\", \"Mail exchange record\",\r\n    QueryType == \"NS\", \"Name server record\",\r\n    QueryType == \"PTR\", \"Pointer record\",\r\n    QueryType == \"SIG\", \"Signature\",\r\n    QueryType == \"SOA\", \"Start of authority�record\",\r\n    QueryType == \"SRV\", \"Service locator\",\r\n    QueryType == \"TXT\", \"Text record\",\r\n    strcat('Other'))\r\n| summarize count()  by type\r\n",
+        "query": "DnsEvents\r\n| extend IPAddresses = iif(IPAddresses==\"\", \"empty\", IPAddresses)\r\n| where \"{IpAddresses:label}\" == \"All\" or IPAddresses in ({IpAddresses})\r\n| extend type = case(\r\n    QueryType == \"A\", \"IPv4 address record\", \r\n    QueryType == \"AAAA\", \"IPv6 address record\",\r\n    QueryType == \"ANY\", \"All cached records\",\r\n    QueryType == \"CNAME\", \"Canonical name record\",\r\n    QueryType == \"MX\", \"Mail exchange record\",\r\n    QueryType == \"NS\", \"Name server record\",\r\n    QueryType == \"PTR\", \"Pointer record\",\r\n    QueryType == \"SIG\", \"Signature\",\r\n    QueryType == \"SOA\", \"Start of authority�record\",\r\n    QueryType == \"SRV\", \"Service locator\",\r\n    QueryType == \"TXT\", \"Text record\",\r\n    strcat('Other'))\r\n| summarize count()  by type\r\n",

```

Comment:
It looks like maybe there is an invalid character here?

Reply:
I am abandoning this because the changes here are consisted in this CR:
[PR](https://github.com/Azure/Azure-Sentinel/pull/357)